index.js

```diff
@@ -0,0 +1,217 @@
+  Docker: 'Docker',
+  Public: 'Public',
+}
+
+const Networks = Object.values(Network)
+
+// ============================================================================
+// Consumer to Platform mapping
+// ============================================================================
+
+/** @type {Record<string, string>} */
+const ConsumerPlatform = {
+  checkin: Platform.Deno,
+  cities: Platform.Deno,
+  files: Platform.Node,
+  games: Platform.Deno,
+  hub: Platform.NextJs,
+  integrations: Platform.Node,
+  landing: Platform.NextJs,
+  locations: Platform.Deno,
+  procedures: Platform.Node,
+  registrations: Platform.Deno,
+  roles: Platform.Deno,
+  themes: Platform.Deno,
+  updates: Platform.Node,
+  users: Platform.Deno,
+  venues: Platform.Deno,
+  vkma: Platform.Vite,
+}
+
+const Consumers = Object.keys(ConsumerPlatform)
+
+// ============================================================================
+// Environment variable access generators per platform
+// ============================================================================
+
+/**
+ * Generates env variable name based on service, network and platform
+ * @param {string} service - Service name (e.g., 'Users')
+ * @param {string} network - Network type (e.g., 'Docker')
+ * @param {string} platform - Target platform
+ * @returns {string} - Environment variable name
+ */
+const generateEnvVarName = (service, network, platform) => {
+  const baseVar = `${service.toUpperCase()}_${network.toUpperCase()}_ORIGIN`
+
+  switch (platform) {
+    case Platform.Vite:
+      return `VITE_${baseVar}`
+    case Platform.NextJs:
+      return `NEXT_PUBLIC_${baseVar}`
+    default:
+      return baseVar
+  }
+}
+
+/**
+ * Generates env variable access code based on platform
+ * @param {string} varName - Environment variable name
+ * @param {string} platform - Target platform
+ * @returns {string} - Code to access the env variable
+ */
+const generateEnvAccess = (varName, platform) => {
+  switch (platform) {
+    case Platform.Deno:
+      return `Deno.env.get('${varName}')`
+    case Platform.Vite:
+      return `import.meta.env.${varName}`
+    case Platform.Node:
+    case Platform.Bun:
+    case Platform.NextJs:
+    default:
+      return `process.env.${varName}`
+  }
+}
+
+// ============================================================================
+// Code generation helpers
+// ============================================================================
+
+/**
+ * Generates ServiceNetworkOrigin object entries
+ * @param {string} platform - Target platform
+ * @returns {string} - Generated code for ServiceNetworkOrigin object
+ */
+const generateServiceNetworkOriginEntries = platform =>
+  Services.map(service => {
+    const networkEntries = Networks.map(network => {
+      const varName = generateEnvVarName(service, network, platform)
+      const envAccess = generateEnvAccess(varName, platform)
+      return `    [Network['${network}']]: ${envAccess},`
+    }).join('\n')
```

Comment:
Try to remove extra spaces from all template strings like this, we can use formatter after generation

Also, you can check this approach before implementing